classes/migration/upgrade/PKPv3_3_0UpgradeMigration.inc.php

```diff
@@ -44,6 +44,10 @@ public function up() {
 			// pkp/pkp-lib#5865 Change announcement expiry format in database
 			$table->date('date_expire')->change();
 		});
+		Capsule::schema()->table('sections', function (Blueprint $table) {
+			$table->tinyInteger('is_inactive')->default(0);
+		});
```

Comment:
Sorry I'm just noticing this now, but I think this migration needs to be moved into each application, because OMP does not have a `sections` table. Instead it has a `series` table.